test/analysis/SmartDetectorsAnalysisTests/SmartDetectorRunnerTests.cs

```diff
@@ -320,5 +338,15 @@ public TestAlert(ResourceIdentifier resourceIdentifier)
             [AlertPresentationProperty(AlertPresentationSection.Property, "Summary title", InfoBalloon = "Summary info")]
             public string Summary { get; } = "Summary value";
         }
+
+        public sealed class TestSmartDetectorDisposable : TestSmartDetector, IDisposable
+        {
+            public bool Disposed { get; private set; }
```

Comment:
>Disposed [](start = 24, length = 8)

WasDisposed

Reply:
and initialize with true in the ctor

---
In reply to: [232665250](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/21#discussion_r232665250) [](ancestors = 232665250)

Reply:
ctor = constructor, right?

---
In reply to: [232665330](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/21#discussion_r232665330) [](ancestors = 232665330,232665250)